framework/SupervisedLearning/DynamicModeDecomposition.py

```diff
@@ -312,10 +368,467 @@ def __returnInitialParametersLocal__(self):
     return self.dmdParams
 
   def __returnCurrentSettingLocal__(self):
+    # print("__returnCurrentSettingLocal__")
     """
       This method is used to pass the set of parameters of the ROM that can change during simulation
       @ In, None
       @ Out, self.dmdParams, dict, the dict of the SM settings
     """
     return self.dmdParams
 
+
+class DynamicModeDecompositionControl(DynamicModeDecomposition):
+  """
+    This surrogate is aimed to construct a "time-dep" surrogate based on
+    Dynamic Mode Decomposition with conmethod.
+    Ref. Proctor, Joshua L., Steven L. Brunton, and J. Nathan Kutz.
+        "Dynamic mode decomposition with control."
+        SIAM Journal on Applied Dynamical Systems 15, no. 1 (2016): 142-161.
+
```

Comment:
A brief description of the dynamic mode decomposition with control would be useful.

Reply:
Also, no documentation has been added to the user manual, which should have the inputs described and a brief description of the method.